optimum/exporters/onnx/base.py

```diff
@@ -303,6 +303,56 @@ def flatten_output_collection_property(cls, name: str, field: Iterable[Any]) ->
         """
         return {f"{name}.{idx}": item for idx, item in enumerate(itertools.chain.from_iterable(field))}
 
+    def generate_dummy_inputs_onnxruntime(self, reference_model_inputs: Mapping[str, Any]) -> Mapping[str, Any]:
```

Comment:
Discuss with lewtun regarding the use of the function. https://github.com/huggingface/transformers/pull/19525#discussion_r1007728512

Reply:
It is needed to generate the inputs for the separate encoder and decoder models?
Is it only used for validation?

Reply:
It is needed only for validation using onnxruntime.  Since the onnx model and torch model will have different input signatures when using `encoder_outputs` for exporting the model.

Reply:
What about calling it `generate_dummy_inputs_for_validation`?
I think it can be misleading otherwise (more so now that we use a `for-ort` argument that does not mean the same thing)